spec/models/evidence_summary_search_form_spec.rb

```diff
@@ -0,0 +1,51 @@
+RSpec.describe EvidenceSummarySearchForm do
+  subject(:search_form) { described_class.new }
+
+  describe '#chosen_options' do
+    context 'when an option has been chosen' do
+      it 'returns true' do
+        search_form.years_of_publication = ['All years', 'Last 2 years']
+        expect(search_form.chosen_options?(
+                 'Last 2 years', :years_of_publication
+        )).to be_truthy
+      end
+    end
+
+    context 'when no options have been chosen' do
+      it 'returns false' do
+        expect(search_form.chosen_options?(
+                 'All years', :years_of_publication
```

Comment:
Identation? Did rubocop catch this line?

Reply:
as above, per the mas standards